presto-thrift-connector/src/main/java/com/facebook/presto/connector/thrift/ThriftConnectorIndex.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.connector.thrift;
+
+import com.facebook.presto.connector.thrift.clientproviders.PrestoThriftServiceProvider;
+import com.facebook.presto.spi.ColumnHandle;
+import com.facebook.presto.spi.ConnectorIndex;
+import com.facebook.presto.spi.ConnectorPageSource;
+import com.facebook.presto.spi.RecordSet;
+
+import java.util.List;
+
+import static java.util.Objects.requireNonNull;
+
+public class ThriftConnectorIndex
```

Comment:
Add a `toString`?

Reply:
I see other implementations of connector index (tpch and Calypso) are not providing toString implementation. Do I still need it?

Reply:
I would expect it will make debugging easier if you expose the `indexHandle`. Other wise when debugging you have to drill in to see what the object is.  IMO the other ones should have a `toString` also

Reply:
I think it shouldn't be logged as part of query plan. In any case, I added `toString` method with `indexHandle`, `lookupColumns` and `outputColumns`.